web/src/api/getState.ts

```diff
@@ -22,32 +32,52 @@ const getState = async (timeAverage: string): Promise<GridState> => {
 };
 
 const useGetState = (): UseQueryResult<GridState> => {
-  const { urlTimeAverage } = useParams<RouteParameters>();
+  const queryClient = useQueryClient();
+  const { urlTimeAverage, urlDatetime } = useParams<RouteParameters>();
+  const isHourly = urlTimeAverage
+    ? URL_TO_TIME_AVERAGE[urlTimeAverage] === TimeAverages.HOURLY
```

Comment:
I think this can just be `const isHourly = URL_TO_TIME_AVERAGE[urlTimeAverage] === TimeAverages.HOURLY`